### ISSUE TITLE
loading version history keep running

### ISSUE THREAD

**Issue by benben994** [ONLYOFFICE/DocumentServer#469]
I've followed up the page - [Document history](https://api.onlyoffice.com/editors/history), but when I press Version History couldn't get any response. In the meantime, I saw that mentioned in #177, the free edition doesn't support the version history feature. Is that true? Or I've making some error?
I've built document server on my own server, and try to deal with the storage service.
![2019-01-07 4 44 56](https://user-images.githubusercontent.com/17497586/50757903-9de87980-129b-11e9-938a-6ec934563dcb.png)

Btw, I've read the comparison between editions, but the comparison is between community edition. I can't find the list of features for document server only.

**Comment by benben994** [comment]
Another short question(In case of the free edition certainly does not support version history)...

If I only need to build document server without community server, which edition should I consider? Do community edition and enterprise edition both have community server?

**Comment by ShockwaveNN** [comment]
@benben994 Hi, Community Editon support Version History by sdk, but #177 mention that CommunityEdition is not best way to test Version History, since there is no integration example in free version and also we do not support Nextcloud\Owncloud for version history. So it's better to check Developer\Integration edition to see how `Version History` work.

> Btw, I've read the comparison between editions, but the comparison is between community edition. I can't find the list of features for document server only.

I don't quite understand what you mean. You can find list of features [here](https://github.com/ONLYOFFICE/DocumentServer#onlyoffice-document-server-editions)

> If I only need to build document server without community server, which edition should I consider?

You can only build Community Edition of DocumentServer from source

**Comment by benben994** [comment]
Hi @ShockwaveNN, sorry for late reply. 
I have figured out that the PHP example from the documentation doesn't implement Version History, but the NodeJS example does. Anyway, I have moved on to implement this part by myself.

Besides, for my understanding from the feature list, "Simultaneous connections" means the users co-editing with same document. But what about "Number of users"? Also it says up to 20 "recommended", can I have more detailed explanation for this?
